components/annot-explanation.vue

```diff
@@ -47,11 +42,11 @@
 
             return {
 
-                // Text for UI elements
-                uiText: {
+                cardTitle: "Explanation",
```

Comment:
This is cleaner, but inconsistent with how we've organized UI text in other components - the strings of which are currently contained in a `uiText` object.

Unless we are planning on moving these strings to the store, I suggest going back to `uiText.cardTitle`.

Reply:
you're right. Makes sense to me